package.json

```diff
@@ -103,10 +103,10 @@
     "qunit-dom": "^1.6.0",
     "semantic-release": "^17.4.4",
     "typescript": "^4.3.4",
-    "xstate": "^4.20.0"
+    "xstate": "^4.20.1"
   },
   "peerDependencies": {
-    "xstate": "^4.20.0"
+    "xstate": "^4.20.1"
```

Comment:
I'm pretty sure updating a peerDependency requirement is a breaking SemVer change.